### PR TITLE
Add template for portfolio, experience and about me info

### PR DESCRIPTION
**Pull request by yicyao**
added header, more info about me, picture and css styling templates, slideshow for project section

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,193 @@
...
+
+                    <div class="column">
+                        <div class = "subheader">HeatSeek</div>
+                        <hr align="center" width="80%">
+                        <div class="about">Hasdf.</div>   
+                    </div> 
+                </div>
+            </div>
+
+
```
Comment (hlz03) [inline portfolio/src/main/webapp/index.html]: remove empty line. Leave only one empty line.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,17 +12,34 @@
...
+  showSlides(slideIndex += n);
 }
+
+// Thumbnail image controls
+function currentSlide(n) {
+  showSlides(slideIndex = n);
+}
+
+function showSlides(n) {
+  var i;
```
Comment (hlz03) [inline portfolio/src/main/webapp/script.js]: use for (let i=0, ...) instead.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,115 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+html, body {
+  overflow-x: hidden;
```
Comment (hlz03) [inline portfolio/src/main/webapp/style.css]: Reorder to be alphabetical order. ditto to the lines below.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -12,17 +12,34 @@
...
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
 }
+
+// Thumbnail image controls
+function currentSlide(n) {
+  showSlides(slideIndex = n);
+}
+
```
Comment (hlz03) [inline portfolio/src/main/webapp/script.js:30]: Add a doc, describing the function, like what's 'n' suppose to mean? Also change n to a more meaningful name.

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -12,17 +12,34 @@
...
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/* Creates Slideshow Carousel to go through my projects*/
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+var slideIndex = 1;
+showSlides(slideIndex += n);
```
Comment (liangqigong) [inline portfolio/src/main/webapp/script.js:18]: Sorry I am new to JS, I don't understand why is this needed?

Comment (liangqigong) [inline portfolio/src/main/webapp/script.js]: Sorry I am new to JS so my question could be dump. Why is this needed?

File: portfolio/src/main/webapp/script.js [hunk 6]
```diff
@@ -12,17 +12,34 @@
...
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+// Next/previous controls
+function plusSlides(n) {
```
Comment (liangqigong) [inline portfolio/src/main/webapp/script.js]: The n here should also be change.

File: portfolio/src/main/webapp/script.js [hunk 7]
```diff
@@ -12,17 +12,34 @@
...
+  showSlides(slideIndex += n);
 }
+
+// Thumbnail image controls
```
Comment (liangqigong) [inline portfolio/src/main/webapp/script.js]: I think the comment here is not clear? 'control' is vague for me.

File: portfolio/src/main/webapp/script.js (hunk 5 above)
Comment (yicyao) [reply portfolio/src/main/webapp/script.js]: no you're right!

**Comment by yicyao** [comment]
Sorry I haven't gotten the hang of rebase yet! This should've been a new PR but my other PR was based on master and not this branch. I have figured it out now and will not do again.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (yicyao) [reply portfolio/src/main/webapp/index.html]: Done.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (yicyao) [reply portfolio/src/main/webapp/script.js]: Done.

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (yicyao) [reply portfolio/src/main/webapp/script.js:30]: I added @param for n (which I changed to slideNum) -- is that what you mean by 'add a doc' @hlz03

File: portfolio/src/main/webapp/script.js (hunk 5 above)
Comment (yicyao) [reply portfolio/src/main/webapp/script.js]: Done

File: portfolio/src/main/webapp/script.js (hunk 7 above)
Comment (yicyao) [reply portfolio/src/main/webapp/script.js]: Done

File: portfolio/src/main/webapp/script.js (hunk 6 above)
Comment (yicyao) [reply portfolio/src/main/webapp/script.js]: Done

File: portfolio/src/main/webapp/index.html [hunk 8]
```diff
@@ -2,17 +2,193 @@
...
+    <!-- Header -->
+    <div class = "header"> Yichen Yao</div>
+    <hr align="center" width="50%">
+
+    <!-- Snap Scroll Start -->
+    <article class="scroller">
+      <!-- Introduction -->
+      <section>
+        <div class="row">
+            <div class="column">
```
Comment (hlz03) [inline portfolio/src/main/webapp/index.html]: nit: Unify the indent to be 2 spaces.
https://google.github.io/styleguide/htmlcssguide.html#Indentation

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (hlz03) [reply portfolio/src/main/webapp/script.js:30]: Yes, you can refer https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments
for js style guide. 

In short, the variable name should be meaningful.

File: portfolio/src/main/webapp/script.js [hunk 9]
```diff
@@ -12,17 +12,48 @@
...
+* @param slideNum: Index of Slide to navigate to
+*/
+
+// Navigate to next slide
+function plusSlides(slideNum) {
+  showSlides("arrow", slideIndex += slideNum);
+
 }
+
+// Navigate to slide indicated by slideNum
```
Comment (hlz03) [inline portfolio/src/main/webapp/script.js]: nit: please change the function level comments align with the style guide. e.g. Navigate -> Navigates.
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

**Review by hlz03 — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html [hunk 10]
```diff
@@ -2,17 +2,211 @@
...
+
+            <div class="column">
+                <div class="about">Hi, I'm Yichen.</div>   
+                <br>
+                <div class="about">Originally from Urbana, IL, I am a rising junior at <a href="https://www.columbia.edu/">Columbia University</a> studying computer science. </div>
+                <br>
+                <div class="about">In my free time, I love to go to concerts, check out new restaurants, and run.</div>
+                <br>
+                <br>
+              <div class="about">Currently reading <a href="https://www.goodreads.com/book/show/52163147-the-hidden-girl-and-other-stories">The Hidden Girl</a>, watching <a href="https://www.imdb.com/title/tt5788792/">The Marvelous Mrs Maisel</a>, learning to play tennis.</div>
```
Comment (liangqigong) [inline portfolio/src/main/webapp/index.html]: The line is pretty long, is there any way to solve it?

I am wondering if you are using any kind of html/css auto formatter? Since Hailun commented on the indentation, I guess a formatting tool could solve both of the issues.

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (yicyao) [reply portfolio/src/main/webapp/style.css]: done.

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (yicyao) [reply portfolio/src/main/webapp/script.js:30]: Done.

File: portfolio/src/main/webapp/index.html (hunk 8 above)
Comment (yicyao) [reply portfolio/src/main/webapp/index.html]: done.

File: portfolio/src/main/webapp/script.js (hunk 9 above)
Comment (yicyao) [reply portfolio/src/main/webapp/script.js]: done

File: portfolio/src/main/webapp/index.html (hunk 10 above)
Comment (yicyao) [reply portfolio/src/main/webapp/index.html]: done.

**Review by hlz03 — APPROVED** [review]
